Test/DafnyTests/TestAttribute.dfy

```diff
@@ -114,7 +114,7 @@ class Even {
   static method AStaticMethod() {}
 }
 
-method {:synthesize} mockUnsafe() 
+method {:synthesize "mock"} mockUnsafe() 
```

Comment:
We can revert all these "mock" parameters now, correct?

Reply:
Yes!